tests/brianqc/linear-dependence/input.dat

```diff
@@ -0,0 +1,64 @@
+#! Water with modified sto-3g to test linearly-dependent basis
```

Comment:
Sure, I will change the number of threads. How many threads would be preferable? Maybe 4 or 2? Or should this also be a single-thread testcase like the others?

My only concern is that when actually running the test suite with BrianQC (as opposed to running them as "dummy" tests with pure Psi4), it would be very unusual for the machine to have enough GPUs for multiple Psi4+BrianQC instances to run in parallel. And even then, the user would need to configure the BrianQC instances to only use one GPU per instance (by default, BrianQC tries to allocate all available GPUs it can handle). So I've originally set the testcase to use 8 threads because BrianQC's tests usually need to be run serially anyway.

Reply:
good point. I would say it should be single thread to fit with running the whole suite. but the `tests/brianqc/` directory will only be added if brianqc is detected at build-time, at which point it's reasonable for the user to manage two-phase testing of `ctest -L brianqc -j1` and `ctest -LE brianqc -j12`. then may as well leave the 8 threads as is so the former runs in reasonable time.